admin/views/tabs/metas/post-types.php

```diff
@@ -14,54 +14,31 @@
  * Otherwise setting a post-type to `noindex` will remove it from the list,
  * making it very hard to restore the setting again.
  */
-$post_types          = get_post_types( array( 'public' => true ), 'objects' );
-$index_switch_values = array(
-	'off' => 'Yes',
-	'on'  => 'No',
-);
+$post_types = get_post_types( array( 'public' => true ), 'objects' );
```

Comment:
Maybe it is nice to use `WPSEO_Post_Type::filter_attachment_post_type( $post_types );` here